spec/models/social_networking/shared_item_spec.rb

```diff
@@ -9,5 +9,20 @@ module SocialNetworking
       shared_item = SharedItem.create(item: goal, action_type: "action_type")
       expect(shared_item.participant_id).to eq(700_141_617)
     end
+
+    describe ".before_save" do
+      let(:activity) do
+        Activity.create!(participant_id: participants(:participant1).id)
+      end
+
+      describe ".set_action_type" do
```

Comment:
@michaelwehrley instance methods typically are denoted with `#` as opposed to `.`.